README.md

```diff
@@ -143,11 +159,25 @@ discouraged).
 '' = 'blank'     # VALID but discouraged
 ```
 
-Values must be of the following types: String, Integer, Float, Boolean,
-Datetime, Array, or Inline Table. Unspecified values are invalid.
+**Dotted keys** are a sequence of bare or quoted keys joined with a dot. This
+allows for grouping similar properties together:
 
 ```toml
-key = # INVALID
+name = "Orange"
+physical.color = "orange"
+physical.shape = "round"
+```
+
+In JSON land, that would give you the following structure:
+
+```json
+{
+  "name": "Orange",
+  "physical": {
+    "color": "orange",
+    "shape": "round"
+  }
+}
```

Comment:
I think this needs a line talking about whitespace and a section about when it's invalid.

@mojombo Something came up just now so I'm probably gonna be AWOL today so, I'm cool with you taking this into your own hands if you want. :)